### ISSUE TITLE
cogvlm模型微调报错（使用Captcha Images数据集）

### ISSUE THREAD

**Issue by FZGgithub** [THUDM/CogVLM#394]
### System Info / 系統信息

系统：
centos8,
cuda:12.1
GPU:A6000*4

### Information / 问题信息

- [ ] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

在完成环境配置后，模型加载完成之后，报错信息如下：
File "/root/anaconda3/lib/python3.11/site-packages/torch/distributed/distributed_c10d.py",line 3439,in barrier
work=default_pg.barrier(opts=opts)
torch.distributed.DistBackendError:NCCL error in :../torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp:1691,unhandled system error (run with NCLL_DEBUG=INFO for details),NCCL version 2.19.3
ncclSystemError:System call (e.g. socket,malloc) or external library call failed or device error.
Last error:
Error while creating shared memory segment /dev/shm/nccl-ENPWQG (size 4194656)
其中，我的shell脚本i信息为：
#! /bin/bash
# export PATH=/usr/local/cuda/bin:$PATH
# export LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH

NUM_GPUS_PER_WORKER=4
MP_SIZE=1

script_path=$(realpath $0)
script_dir=$(dirname $script_path)
main_dir=$(dirname $script_dir)
MODEL_TYPE="/root/CogVLM/cogvlm-chat-v1.1"
VERSION="base"
MODEL_ARGS="--from_pretrained $MODEL_TYPE \
    --max_length 1288 \
    --lora_rank 10 \
    --use_lora \
    --local_tokenizer /root/CogVLM/vicuna-7b-v1.5 \
    --version $VERSION"
# Tips: If training models of resolution 244, you can set --max_length smaller 

OPTIONS_SAT="SAT_HOME=~/.sat_models"
OPTIONS_NCCL="NCCL_DEBUG=info NCCL_IB_DISABLE=0 NCCL_NET_GDR_LEVEL=2 LOCAL_WORLD_SIZE=$NUM_GPUS_PER_WORKER"
HOST_FILE_PATH="hostfile"

train_data="./archive_split/train"
valid_data="./archive_split/valid"

gpt_options=" \
       --experiment-name finetune-$MODEL_TYPE \
       --model-parallel-size ${MP_SIZE} \
       --mode finetune \
       --train-iters 800 \
       --resume-dataloader \
       $MODEL_ARGS \
       --train-data ${train_data} \
       --valid-data ${valid_data} \
       --distributed-backend nccl \
       --lr-decay-style cosine \
       --warmup .02 \
       --checkpoint-activations \
       --vit_checkpoint_activations \
       --save-interval 200 \
       --eval-interval 200 \
       --save "./checkpoints" \
       --eval-iters 10 \
       --eval-batch-size 1 \
       --split 1. \
       --deepspeed_config test_config_bf16.json \
       --skip-init \
       --seed 2023
"
run_cmd="${OPTIONS_NCCL} ${OPTIONS_SAT} deepspeed --master_port 16666 --hostfile ${HOST_FILE_PATH} finetune_cogvlm_demo.py ${gpt_options}"
echo ${run_cmd}
eval ${run_cmd}

set +x

请问我该如何解决这个问题呢

### Expected behavior / 期待表现

因为是在不联网环境里进行配置的，所以不知道是否是相关依赖有所问题

**Comment by elesun2018** [comment]
NCCL error 
 run with NCLL_DEBUG=INFO for details
Error while creating shared memory segment /dev/shm
我认为是NCCL配置的的问题，采用NCLL_DEBUG=INFO查看具体原因，
 shared memory的问题？查看英伟达官网NCCL环境变量配置
https://docs.nvidia.com/deeplearning/nccl/user-guide/docs/env.html

**Comment by FZGgithub** [comment]
经过对docker容器的共享内存的重分配，上述问题已经解决。但是在运行finetune_cogvlm_lora.sh脚本时，会频繁打印一句提示:Keyword arguments {'add_special_tokens':False} not recognized
我并未在代码里面找到相关的代码，请问这个错误是模型选择的问题吗？（我目前使用的模型是sat版本的cogvlm-chat-v1.1）
